pymc3/distributions/simulator.py

```diff
@@ -135,22 +136,31 @@ def identity(x):
     return x
 
 
-def gaussian_kernel(epsilon, obs_data, sim_data):
-    """gaussian distance function"""
-    return np.sum(-0.5 * ((obs_data - sim_data) / epsilon) ** 2)
+def gaussian(epsilon, obs_data, sim_data):
+    """Gaussian kernel distance function."""
```

Comment:
Is it Gaussian distance or Gaussian distance kernel? The docstring and function name seem to disagree

Reply:
Good you ask!  I am using "distance" in a sloppy broad sense. So with this docstring I am trying to said that this particular distance (function) is a "Gaussian kernel". 

I know this can be confusing let me explain, many ABC methods use distances to define which samples to keep and which ones to reject. The rejected are those that are larger than a cutoff value. Thus the name "distance" or "distance function" is widespread. But in PyMC3 we do not use a hard threshold, instead we use a "kernel" to measure a continuous similarity function. Thus we replace the likelihood (that we can not compute) by a function of the distance between observed and simulated data.

Reply:
Ah ok, I think I understood, thanks for the clear explanantion!
Shouldn't the function be named `gaussian_kernel` then, for clarity?

Reply:
maybe (that was its previous name), but that's too long. The function name is used for the str/latex representation of the model and also I would like to keep the name of the function the same as the name of the argument passed to distance.

Reply:
Ah ok, I see. I'll let you be the sole judge here then --> approving 🎉